lib/views/request/request_view.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:clnapp/api/api.dart';
+import 'package:clnapp/views/pay/qr_screen.dart';
+import 'package:flutter/material.dart';
+import '../../components/buttons.dart';
+import '../../model/app_model/generate_invoice.dart';
+import '../../utils/app_provider.dart';
+
+class RequestView extends StatefulWidget {
+  final AppProvider provider;
+
+  const RequestView({Key? key, required this.provider}) : super(key: key);
+
+  @override
+  State<RequestView> createState() => _RequestViewState();
+}
+
+class _RequestViewState extends State<RequestView> {
+  String display = '';
```

Comment:
```suggestion
  String? display;
```

let the compiler help you